python/README.md

```diff
@@ -1,58 +1,58 @@
-You can also run this locally by installing the [stocator driver](https://github.com/SparkTC/stocator)
-and adding it to your local Apache Spark kernel's classpath.
 
 ## Installation
 
-This library is now installed by default on IBM Apache Spark.
+This library is published on PyPI and may be installed with `pip`.
 
 ```
-pip install --user --upgrade ibmos2spark
+pip install ibmos2spark
 ```
 
-## Usage
+## Apache Spark at IBM
 
-The usage of this package depends on *from where* your Object Storage instance was created. This package
-is intended to connect to IBM's Object Storage instances (Swift OS). This OS can be obtained from Bluemix or Data Science Experience (DSX) or from a separate account on IBM Softlayer. The package also supports IBM Cloud Object Storage as well (COS).
-The instructions below show how to connect to either type of instance.
+The `stocator` and `ibmos2spark` libraries are pre-installled and available on 
 
-The connection setup is essentially the same. But the difference for you is how you deliver the
-credentials. If your Object Storage was created with Bluemix/DSX, with a few clicks on the side-tab
-within a DSX Jupyter notebook, you can obtain your account credentials in the form of a Python dictionary.
-If your Object Storage was created with a Softlayer account, each part of the credentials will
-be found as text that you can copy and paste into the example code below.
+* [Apache Spark through IBM Bluemix](https://console.bluemix.net/catalog/services/apache-spark)
+* [IBM Analytics Engine (Beta)](https://console.bluemix.net/catalog/services/ibm-analytics-engine)  
+* [IBM Data Science Experience](https://datascience.ibm.com)
 
-### Softlayer IBM Cloud Object Storage (COS)
-```python
-import ibmos2spark
 
-credentials = {
-  'endpoint': 'https://s3-api.objectstorage.softlayer.net/',  #just an example. Your url might be different
-  'access_key': '',
-  'secret_key': ''
-}
 
-configuration_name = 'cos_config_string'  #you can give any string you like
-cos = ibmos2spark.CloudObjectStorage(sc, credentials, configuration_name=configuration_name)  #sc is the SparkContext instance.
+## Usage
+
+The instructions below demonstrate how to use this package to retrieve data from the various 
+IBM Object Storage services.
+
+These instructions will refer to the image at the top of this README.
 
-bucket_name = 'some_bucket_name'
-object_name = 'file1'
-data = sc.textFile(cos.url(object_name, bucket_name))
-```
 
-### Bluemix IBM Cloud Object Storage (COS)
-The class CloudObjectStorage allows you to connect to an IBM bluemix COS. You can connect to a bluemix COS using api keys
-as follows:
+### Cloud Object Storage 
+
+This is the service described on the **far left** in the image above. This service is also called IBM Bluemix Cloud Object Storage (COS) in various locations. 
+
+A `configuration_name` is required and the `cos_type` keyword argument must be set to `bluemix_cos`.
```

Comment:
Currently, configuration_name is not required. if we change this in a future PR we will update the readme file accordingly on that PR.

Reply:
done.